### PR TITLE
SUBMARINE-550. [doc] Upload pysubmarine to pypi

### PR DESCRIPTION
**Pull request by pingsutw**
### What is this PR for?
The doc including how to upload python package to pypi

Note:
I've uploaded pysubmarine package to **testPypi** for testing
https://test.pypi.org/project/submarine-sdk/
`pip install -i https://test.pypi.org/simple/ submarine-sdk
`

After review, I will create an account and push the package to **Pypi**

After discuss offline, The final name will be `apache-submarine`

### What type of PR is it?
[Documentation]

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-550

### How should this be tested?
https://travis-ci.org/github/pingsutw/hadoop-submarine/builds/703397771

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: submarine-sdk/pysubmarine/README.md [hunk 1]
```diff
@@ -21,39 +21,26 @@ tracking experiment metrics, parameters.
...
-```
-- Run checkstyle
-```bash
-./submarine-sdk/pysubmarine/github-actions/lint.sh
-```
-## How to generate REST SDK from swagger
+- Install package from pypi
 ```bash
-./dev-support/pysubmarine/gen-sdk.sh
+pip install submarine-sdk
```
Comment (jiwq) [inline submarine-sdk/pysubmarine/README.md:32]:
```suggestion
pip install apache-submarine
```

File: submarine-sdk/pysubmarine/setup.py [hunk 2]
```diff
@@ -15,11 +15,15 @@
 
 from setuptools import setup, find_packages
 
+with open('README.md') as f:
+    long_description = f.read()
 
 setup(
-    name='pysubmarine',
+    name='submarine-sdk',
```
Comment (jiwq) [inline submarine-sdk/pysubmarine/setup.py:22]: `PySubmarine` or `apache-submarine` ? I think it's not the `submarine-sdk`.

File: submarine-sdk/pysubmarine/setup.py [hunk 3]
```diff
@@ -43,4 +47,7 @@
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
     ],
+    license='Apache License, Version 2.0',
+    maintainer='Apache Submarine Developers',
```
Comment (jiwq) [inline submarine-sdk/pysubmarine/setup.py:51]:
```suggestion
    maintainer='Apache Submarine Community',
```

File: submarine-sdk/pysubmarine/setup.py (hunk 2 above)
Comment (pingsutw) [reply submarine-sdk/pysubmarine/setup.py]: I've updated it

**Comment by pingsutw** [comment]
upload to [TestPypi](https://test.pypi.org/project/apache-submarine/) for testing

**Comment by pingsutw** [comment]
@liuxunorg, it could be merged if there is no problem in this [page](https://test.pypi.org/project/apache-submarine/)
could you help me take a look?
After the PR be merged, I will create an account and push this python package to **PyPI**
so that users could download by `pip install apache-submarine`

**Comment by jiwq** [comment]
@pingsutw I'm sorry this PR conflict caused by the preparing 0.5.0 development. Can you help to fix it?

**Comment by pingsutw** [comment]
updated it.

**Comment by liuxunorg** [comment]
> @liuxunorg, it could be merged if there is no problem in this [page](https://test.pypi.org/project/apache-submarine/)
> could you help me take a look?
> After the PR be merged, I will create an account and push this python package to **PyPI**
> so that users could download by `pip install apache-submarine`

LGTM

**Comment by liuxunorg** [comment]
Will merge if no more comments.

**Pull request closed without merging**